README.md

```diff
@@ -1 +1,253 @@
+- [2] **Integration of a search functionality**
+- [3] **Mise en place d'un système d'authentification et d'autorisation pour les utilisateurs**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+CONTRIBUTING
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions are welcome and will be appreciated! If you would like to contribute to this project, please follow these steps:
+
+Fork this project.
+
+Create a branch for your feature > git checkout -b feature/AmazingFeature.
+
+Commit your changes > git commit -m 'Add some AmazingFeature'.
+
+Push your branch > git push origin feature/AmazingFeature.
+
+Open a pull request.
+
+Once your pull request has been reviewed and accepted, your feature will be added to the project.
+
+Feel free to contact me if you have any questions or comments on how to contribute to this project.
+
+
+SUPPORT
+
+## ⭐️ Show your support
+If you liked this project, don't hesitate to support it by giving a ⭐️!
+
+It means a lot to me and encourages me to continue working on this project and to create other interesting projects in the future.
+
+If you have any comments or suggestions to improve this project, do not hesitate to contact me.
+
+If you like this project...
+
+
+ACKNOWLEDGEMENTS <a name="acknowledgements"></a>
+
+## 🙏 Acknowledgments 
+
+I would like to thank NTIHINDUKA ALPHA, ROBERTSON ARTUR for their excellent work in revising my README file. His feedback greatly improved the quality and clarity of my project documentation.
+
+His contribution and support have been of great help to me throughout the development of this project. I am grateful for his time and expertise.
+
+Thanks again, NTIHINDUKA ALPHA!
+
+I am grateful for any input and support for this project. If you have contributed to the project and are not on this list, please contact me so I can add you.
+
+
+## 📝 License 
+
+MIT License
+
+Copyright (c) 2023 GhostEsso
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```

Comment:
- [ ] As the previous reviewer mentioned, You need to create a LICENSE.md file which will contain these details you have here, and in your README file, all you need to include is the link to the file as it was previously and not these details. So you just need to ensure that the link you'll put in your README file takes you to the LICENSE file you create and not to an error page as it did previously. :100: 
- [ ] You can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project :wink:

Reply:
Hi **@NahnahAJ** Thank you for your recommendations... I followed the creation of a LICENCE.md file, but look at the error message when I click on the link from Readme so that it sends me to the LICENSE file there is always a error message! I don't know where it's stuck...  
![image](https://user-images.githubusercontent.com/114746721/219210870-f2c85a8a-1b52-4c49-801a-8e8b61bb58da.png)